content/GettingStarted/SCALEReleaseNotes.md

```diff
@@ -73,43 +126,40 @@ Click the component version number to see the latest release notes for that comp
+| block_cloning | [com.fudosecurity:block_cloning](https://openzfs.github.io/openzfs-docs/man/master/7/zpool-features.7.html#com.fudosecurity:block_cloning) | This is enabled by default for pools created in 23.10 (Cobia). Systems upgrading to 23.10 have to upgrade existing pools to enable this feature. |
+| draid | [org.openzfs:draid](https://openzfs.github.io/openzfs-docs/man/master/7/zpool-features.7.html#org.openzfs:draid) | Web interface support is not present in 23.10-BETA.1. |
 | head_errlog | [com.delphix:head_errlog](https://openzfs.github.io/openzfs-docs/man/master/7/zpool-features.7.html#com.delphix:head_errlog) | |
 | vdev_zaps_v2 | [com.klarasystems:vdev_zaps_v2](https://openzfs.github.io/openzfs-docs/man/master/7/zpool-features.7.html#com.klarasystems:vdev_zaps_v2) | |
-| zilsaxattr | [org.openzfs:zilsaxattr](https://openzfs.github.io/openzfs-docs/man/master/7/zpool-features.7.html#org.openzfs:zilsaxattr) | |
+| zilsaxattr | [org.openzfs:zilsaxattr](https://openzfs.github.io/openzfs-docs/man/master/7/zpool-features.7.html#org.openzfs:zilsaxattr) | Web interface support is not present in 23.10-BETA.1. |
 {{< /truetable >}}
-{{< /expand >}}
 
-## 23.10-BETA.1
-{{< hint type=warning >}}
+For more details on feature flags see [OpenZFS Feature Flags](https://openzfs.github.io/openzfs-docs/Basic%20Concepts/Feature%20Flags.html).
+For more details on zpool.features.7 see [OpenZFS zpool-feature.7](https://openzfs.github.io/openzfs-docs/man/7/zpool-features.7.html).
+
+## 23.10-BETA.1 Changelog
+{{< hint type=warning title="Early Release Software" >}}
 Early releases are intended for testing and early feedback purposes only.
 Do not use early release software for critical tasks.
 {{< /hint >}}
 **August 15, 2023**
 
-iXsystems is pleased to release TrueNAS SCALE 23.10-BETA.1!
-
-### 23.10-BETA.1 Changelog
+<a href="https://ixsystems.atlassian.net/issues/?filter=10359&atlOrigin=eyJpIjoiNTczY2E2NmVjODk5NGE0NThlZTFlOTI4MDFhOTMzNzUiLCJwIjoiaiJ9" target="_blank">Click here</a> for the full changelog of completed tickets that are included in the 23.10-BETA.1 release.
 
-<!-- add copy about logging in to jira to see the list of tickets (test to confirm behavior), add link to jira filter -->
+With the changelog open in Jira, click <span class="iconify" data-icon="mdi:export-variant"></span> **Export** to print or download the changelog in various file formats.
 
-## TrueNAS SCALE Ongoing Issues
+## 23.10 Ongoing Issues
 
-These issues have been discovered through community or internal testing and are being tracked for resolution.
+[Click here](https://ixsystems.atlassian.net/issues/?filter=10357&atlOrigin=eyJpIjoiZGYwODg1MjgzZDFiNDVkN2FlMGRjNzcwZGJjMzRiODkiLCJwIjoiaiJ9) to view the latest information about tracked issues that have been discovered through community or internal testing.
```

Comment:
There is a defect marked iX Private on the linked list.

Also, our old Known Issues had unresolved issues that were not linked to a particular release. If users are looking for status on those issues, or they encounter the Bluefin issues in Cobia, are they covered in the linked lists?

Reply:
The filter link is based on your login - if you use an inprivate browser so your employee creds aren't saved, you shouldn't see anything marked private when clicking the link.
We're still working out the issues that need the label, the main thing to test here is that a browser with no Jira credentials can see the relnotes and issue list without any of the internal notes issues.